### PR TITLE
chore: rename "project show" to "app show"

### PR DESCRIPTION
**Pull request by efekarakus**
*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

File: internal/pkg/cli/app_show.go [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/config"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/describe"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/spf13/cobra"
+)
+
+const (
+	appShowNamePrompt     = "Which application would you like to show?"
+	appShowNameHelpPrompt = "An application groups all of your services together."
```
Comment (kohidave) [inline internal/pkg/cli/app_show.go:20]: > An application is a collection of related services. 

What do you think about that?

Comment (efekarakus) [reply internal/pkg/cli/app_show.go]: I think that's good! updated

Comment (iamhopaul123) [reply internal/pkg/cli/app_show.go]: just a reminder if we decide to make this change then we need to change the other files for consistency

File: internal/pkg/cli/app_show_test.go [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+			mockStoreReader := mocks.NewMockstore(ctrl)
+			mockPrompter := mocks.NewMockprompter(ctrl)
+
+			mocks := showAppMocks{
+				storeSvc: mockStoreReader,
+				prompt:   mockPrompter,
+			}
+			tc.setupMocks(mocks)
+
+			showProjects := &showAppOpts{
```
Comment (iamhopaul123) [inline internal/pkg/cli/app_show_test.go:67]: maybe showApp?

File: internal/pkg/cli/app_show.go (hunk 1 above)
Comment (efekarakus) [reply internal/pkg/cli/app_show.go]: I'll update the others in a separate PR

File: internal/pkg/cli/app_show_test.go (hunk 2 above)
Comment (efekarakus) [reply internal/pkg/cli/app_show_test.go:67]: fixed it here! https://github.com/aws/amazon-ecs-cli-v2/pull/927

**Pull request merged**